[2025-02-02T16:34:00.731+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T16:34:00.741+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.load_data_fly manual__2025-02-02T16:33:54.858191+00:00 [queued]>
[2025-02-02T16:34:00.745+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.load_data_fly manual__2025-02-02T16:33:54.858191+00:00 [queued]>
[2025-02-02T16:34:00.746+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-02-02T16:34:00.754+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): load_data_fly> on 2025-02-02 16:33:54.858191+00:00
[2025-02-02T16:34:00.759+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1000) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T16:34:00.760+0000] {standard_task_runner.py:72} INFO - Started process 1032 to run task
[2025-02-02T16:34:00.760+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'load_data_fly', 'manual__2025-02-02T16:33:54.858191+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmpz86ybs9q']
[2025-02-02T16:34:00.761+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask load_data_fly
[2025-02-02T16:34:00.807+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.load_data_fly manual__2025-02-02T16:33:54.858191+00:00 [running]> on host d628bd12729b
[2025-02-02T16:34:00.915+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='load_data_fly' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T16:33:54.858191+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T16:33:54.858191+00:00'
[2025-02-02T16:34:00.917+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T16:35:15.145+0000] {logging_mixin.py:190} INFO - Data loaded into DuckDB successfully.
[2025-02-02T16:35:15.187+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T16:35:15.195+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T16:35:15.196+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=load_data_fly, run_id=manual__2025-02-02T16:33:54.858191+00:00, execution_date=20250202T163354, start_date=20250202T163400, end_date=20250202T163515
[2025-02-02T16:35:15.228+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T16:35:15.237+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T16:35:15.240+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
